### PR TITLE
Fix (again) verify-owners

### PR DESCRIPTION
**Pull request by matthyx**
Add missing position in github.DraftReviewComment
Improve messages
Should fix https://github.com/kubernetes/test-infra/pull/8430

### CODE REVIEW

File: prow/plugins/verify-owners/verify-owners.go [hunk 1]
```diff
@@ -138,50 +137,51 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
...
 		// Check labels against blacklist
 		if sets.NewString(labels...).HasAny(labelsBlackList...) {
-			wrongOwnersFiles[f] = fmt.Errorf("OWNERS file contains blacklisted labels: %s", sets.NewString(simple.Config.Labels...).Intersection(sets.NewString(labelsBlackList...)).List())
+			wrongOwnersFiles[f] = fmt.Sprintf("File contains blacklisted labels: %s.", sets.NewString(labels...).Intersection(sets.NewString(labelsBlackList...)).List())
 			continue
 		}
 		// Check approvers isn't empty
 		if filepath.Dir(f) == "." && len(approvers) == 0 {
-			wrongOwnersFiles[f] = errors.New("no approvers in the root directory OWNERS file")
+			wrongOwnersFiles[f] = "No approvers defined in file."
```
Comment (cjwagner) [inline prow/plugins/verify-owners/verify-owners.go]: I think it is important to specify that the problem is that the **root** OWNERS file does not have approvers. The new message suggests that all OWNERS files must have approvers defined which is not the case.

File: prow/plugins/verify-owners/verify-owners.go [hunk 2]
```diff
@@ -138,50 +137,51 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
...
-		s := "s"
-		if len(wrongOwnersFiles) == 1 {
-			s = ""
-		}
-		response := fmt.Sprintf("%d invalid OWNERS file%s", len(wrongOwnersFiles), s)
+		response := fmt.Sprintf("%d invalid %s file%s", len(wrongOwnersFiles), ownersFileName, s)
 		err := ghc.CreateReview(org, repo, pre.Number, github.DraftReview{
 			Body:     plugins.FormatResponseRaw(pre.PullRequest.Body, pre.PullRequest.HTMLURL, pre.PullRequest.User.Login, response),
 			Action:   github.Comment,
 			Comments: comments,
```
Comment (cjwagner) [inline prow/plugins/verify-owners/verify-owners.go]: We should also specify the `commit_id` field to ensure that the review is posted for the right commits even if the PR is updated while we are posting the review. https://developer.github.com/v3/pulls/reviews/#input

File: prow/plugins/verify-owners/verify-owners.go (hunk 1 above)
Comment (matthyx) [reply prow/plugins/verify-owners/verify-owners.go]: Indeed, but the rationale was that the comment was to be attached to the proper file... Maybe I will specify that root OWNERS files must have approvers.

Comment (cjwagner) [reply prow/plugins/verify-owners/verify-owners.go]: I think we should still specify that it is the root file to make it clear that the requirement only applies to that file.

**Comment by stevekuznetsov** [comment]
/lgtm
/hold
@cjwagner

**Comment by stevekuznetsov** [comment]
@matthyx looks like you need to update the PR!

**Pull request merged**